config/lorekeeper/extensions.php

```diff
@@ -83,8 +83,16 @@
         'currency_id' => 1,
     ],
 
-    // Organised Traits Dropdown - Draginraptor
-    'organised_traits_dropdown' => 0,
+    // Organised Traits Dropdown - Draginraptor, ScuffedNewt
+    'organised_traits_dropdown' => [
+        'enable'           => 1,
+        'display_species'  => 0, // displays species
+        'display_subtype'  => 0, // displays subtype, SPECIES DISPLAY MUST ALSO BE ENABLED FOR THIS TO WORK.
```

Comment:
```suggestion
        'display_subtype'  => 0, // displays subtype. NOTE: species display must also be enabled for this to work.
```
nit: whole note should probably not be caps (though some emphasis is probably useful)

Reply:
Good point!